sensors/avl/sensorhub-driver-avl-911/build.gradle

```diff
@@ -17,6 +17,11 @@ osgi {
   manifest {
     attributes('Bundle-Vendor': 'Botts Innovative Research, Inc.')
     attributes('Bundle-Activator': 'org.sensorhub.impl.sensor.avl.Activator')
+    attributes('Export-Package': 'org.sensorhub.impl.sensor.avl')
+    attributes('Import-Package': '''
+      !java.io.*,!java.lang.*,!java.net.*,!java.nio.file.*,!java.text.*, !java.time.*,!java.util.*,
+      org.osgi.service.repository;resolution:=optional,
```

Comment:
Don't think we need to import "org.osgi.service.repository;resolution:=optional"

Reply:
No need, Ok for building and installing.  cdc2b03d777c70987963c2bf817b2c32e7410ddf